services/galley/src/Galley/API/LegalHold.hs

```diff
@@ -142,6 +143,14 @@ removeSettings zusr tid (Public.RemoveLegalHoldSettingsRequest mPassword) = do
   void $ permissionCheck ChangeLegalHoldTeamSettings zusrMembership
   ensureReAuthorised zusr mPassword
   removeSettings' tid
+  where
+    assertNotWhitelisting :: Galley ()
```

Comment:
I want to rename this to `assertNoSettingsWhitelist`, but I'll try to control myself.